code/modules/mob/living/silicon/silicon.dm

```diff
@@ -39,15 +39,18 @@
 /mob/living/silicon/New()
 	GLOB.silicon_mob_list |= src
 	..()
-	var/datum/atom_hud/data/diagnostic/diag_hud = GLOB.huds[DATA_HUD_DIAGNOSTIC]
-	diag_hud.add_to_hud(src)
-	diag_hud_set_status()
-	diag_hud_set_health()
 	add_language("Galactic Common")
 	init_subsystems()
 	RegisterSignal(SSalarm, COMSIG_TRIGGERED_ALARM, .proc/alarm_triggered)
 	RegisterSignal(SSalarm, COMSIG_CANCELLED_ALARM, .proc/alarm_cancelled)
 
+/mob/living/silicon/Initialize()
```

Comment:
The HUD list is made during `Initialize()` at `/mob` level, and making decoy AI cores parent call `New()` caused them to get HUDs without it working. This fixes that.